Gemfile.lock

```diff
@@ -188,7 +188,7 @@ GEM
       activesupport (= 4.1.10)
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
-    rake (12.0.0)
+    rake (10.5.0)
```

Comment:
Rake 11.0.1 removes the last_comment method which rspec-core uses. So we have to either add this method in the rake file manually when using updated version or have to use the versions before 11.0